dashboard/config/locales/en.yml

```diff
@@ -22,10 +22,10 @@
 en:
   errors:
     messages:
-      not_saved: 'Please correct the following errors:'
-      too_young: 'This content has age restrictions in place and is not available for younger students.'
+      not_saved: "Please correct the following errors:"
+      too_young: "This content has age restrictions in place and is not available for younger students."
```

Comment:
Is there a reason we want to change the quote convention here?

Reply:
There is not. Sorry about that - I use prettier as a formatter and this particular file gets a bit goofy from that. This file should not be part of this push anymore.